akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -439,6 +442,7 @@ private[akka] class ShardRegion(
     case query: ShardRegionQuery                 ⇒ receiveQuery(query)
     case msg: RestartShard                       ⇒ deliverMessage(msg, sender())
     case msg if extractEntityId.isDefinedAt(msg) ⇒ deliverMessage(msg, sender())
```

Comment:
Another thing, it would be very useful to be able to send `StartEntity` also from the outside, and not only for remembering entities. Then that can be used by a keep-alive service to ping entities periodically (that is a better choice than using rememeber entites if the entity ids are well known). Then the ack message should also be public.